src/screens/SUE/SueMapScreen.tsx

```diff
@@ -156,6 +156,13 @@ export const SueMapScreen = ({ navigation, route }: Props) => {
                 <RegularText smallest>{item.address}</RegularText>
               </View>
               <SueStatus iconName={item.iconName} status={item.status} small />
+
+              <TouchableOpacity
+                onPress={() => setSelectedRequestId(undefined)}
+                style={styles.closeButton}
+              >
+                <Icon.Close size={normalize(20)} />
```

Comment:
how comes this solution that is different to the layouts?

![image](https://github.com/smart-village-solutions/smart-village-app-app/assets/1942953/5db129c0-d646-4d38-adfb-e61646499a64)